src/utils/loadable.ts

```diff
@@ -0,0 +1,52 @@
+import { atom } from 'jotai'
+import type { Atom } from 'jotai'
+
+const loadableAtomCache = new WeakMap<Atom<unknown>, Atom<Loadable<any>>>()
```

Comment:
Can `any` be `unknown`? Or, TS complains?

Reply:
It can be unknown, it'll just require a cast further down. I'll update